manifest.yml

```diff
@@ -0,0 +1,72 @@
+# Manifest for image directories creation
+# every dest path will be prefixed by $DESTDIR/$version
+
+# Files containing distgen directives
+DISTGEN_RULES:
+  - src: src/cccp.yml
+    dest: cccp.yml
```

Comment:
Is `cccp.yml` still used? I am not sure, but it was removed for python container.

Reply:
No idea here but I can remove it, sure.

Reply:
@pkubatrh wdyt?

Reply:
Because the file says, that it is present for the purpose of building the container on the CentOS Container Pipeline. But we build the centos and centos stream containers via GitHub Actions now.

Reply:
Files and their definition removed.